src/convertToObject.js

```diff
@@ -3,16 +3,30 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an example in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const formattedString = sourceString
+    .split(';')
+    .map((string) => string.trim())
+    .filter((string) => string);
+
+  return formattedString.reduce((accumulator, string) => {
+    const parts = string.split(':');
+    const [ part1, part2 ] = parts;
```

Comment:
it would be better to have more meaningful name like `const [key, value] = styleRule`